src/client/main.rs

```diff
@@ -30,6 +30,11 @@ pub fn get_proxy() -> zbus::Result<ServerProxyBlocking<'static>> {
 }
 
 fn main() -> Result<(), glib::Error> {
+	// Parse Config
```

Comment:
Add some config dir information in the client, server, and backends `--help` print information

Reply:
Sounds like a good idea.

Though, I can't find a way to add text to `--help` without adding another option with `gtk-rs`. :sweat_smile: 

I think one way to workaround that would be to add a `--config` option that allows specifying a custom config path, and explain the config stuff in the help text for that option.

I'll look at this at some point in the next few days, when I have a bit more time.

Reply:
> Sounds like a good idea.
> 
> Though, I can't find a way to add text to `--help` without adding another option with `gtk-rs`. :sweat_smile: 
> 
> I think one way to workaround that would be to add a `--config` option that allows specifying a custom config path, and explain the config stuff in the help text for that option.
> 
> I'll look at this at some point in the next few days, when I have a bit more time.

Oh yeah, true. Sounds like a good idea 👍

Reply:
Implemented the `--config` option today. Sorry it took so long. The libinput backend doesn't have any command line parsing at the moment, so I left it out there, but I can add similar parsing to it as with client/server.

The help message isn't final yet, I still need to think about how to best communicate the location without hardcoding paths or making the message too long.

Let me know if you like the basic implementation outline or if you'd like it done differently.